core/handler_type.go

```diff
@@ -0,0 +1,11 @@
+package core
+
+import (
+	"github.com/yomorun/yomo/core/frame"
+)
+
+// SimpleHandler is the request-response mode (asnyc)
+type SimpleHandler func([]byte) (byte, []byte)
```

Comment:
shall we rename `SimpleHandler` to `RawHandler` or `AsyncHandler` ? the signature like `([]byte) -> (byte, []byte)` is way like `http handler`, the `request-response` mode.